upgrade_tests/helpers/existence.py

```diff
@@ -419,8 +419,8 @@ def _find_templatestore(templatestorestate, template_type, template_id=None):
     template_id = template_id.strip()
     template_path = '{0}/{1}.erb'.format(templates_path, template_id)
     if not os.path.exists(template_path):
-        return 'ID : {0} {1} is missing.'.format(
-            template_id, template_type, templatestorestate)
+        return '{0} template of ID {1} is missing'.format(
```

Comment:
return f'{template_type} template of ID {template_id} is missing' instead of 
return '{0} template of ID {1} is missing'.format(template_type, template_id)

Reply:
Yeah, just following the existing method in that function of string formatting, we can f-string in separate PR btw. What do u think ?

Reply:
I mean fstring for all the sat upgrade repo.

Reply:
I think this is a small change and we can handle it here, for repo related changes will take care in the future.

Reply:
Done!